### ISSUE TITLE
how to run a demo

### ISSUE THREAD

**Issue by HuaYuexia** [vidit09/domaingen#30]
Excuse me, can you tell me how to run a demo to test a picture?

**Comment by vidit09** [comment]
Hello 
you can run the following cmd
`python train.py --eval-only --config-file configs/diverse_weather.yaml`

**Comment by HuaYuexia** [comment]
Thanks for your reply! The script above seems to be for testing. But I am still confused about how to test one or some pictures and get their visualized results as shown in the paper. I am very much looking forward to your detailed response!

**Comment by vidit09** [comment]
You pull out the model from this line  
https://github.com/vidit09/domaingen/blob/9c03d07a6f8c12b6a7a5e02bd9346d7bb5a3b9ff/train.py#L579-L582

and pass the image with the required preprocessing

**Comment by HuaYuexia** [comment]
Thank you very much for your patience! I'm a beginner so I'll be a bit confused, can I refer to detectron2's demo.py for the rest of the processing steps? Sorry for wasting your time, thanks again!

**Comment by vidit09** [comment]
Yes, you can try that. I'm not sure how compatible the versions are but if underlying base classes are the same, it can work.

**Comment by Git9853** [comment]
Hello, I have the same questions. Could you please share your demo script? Looking forward to your reply.

> Yes, you can try that. I'm not sure how compatible the versions are but if underlying base classes are the same, it can work.
